### PR TITLE
WPF-61114 Prepared UG content for SmithChart

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
Hi @vigneshrameshsync ,
                   I have prepared UG content for SmithChart.

### CODE REVIEW

File: wpf/Smith-Chart/Series.md [hunk 1]
```diff
@@ -144,3 +144,32 @@ chart.Series.Add(series2);
 {% endtabs %}
 
 ![Series_img3](Series_images/Series_img3.png)
+
+## Customizing Series Data
```
Comment (vigneshrameshsync) [inline wpf/Smith-Chart/Series.md:148]: This is just a heading, not an API. So, please avoid unwanted casing.

File: wpf/Smith-Chart/Series.md [hunk 2]
```diff
@@ -144,3 +144,32 @@ chart.Series.Add(series2);
...
+
+The property *ArrangeByIndex* value is indicating whether to render the data based on the provided index order.the data points are rendered based on the provided index when it is true otherwise, the data points are rendered by sorting based on its resistance value.
```
Comment (vigneshrameshsync) [inline wpf/Smith-Chart/Series.md:150]: API should be provided as a link using the syntax \[\`API_NAME\`\](LINK).

You just provided an API summary as a description. It is wrong, the description should explain the use-case of this API clearly. Please refer to the below links.

https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CategoryAxis/arrangeByIndex.html

https://help.syncfusion.com/wpf/classic/chart/chart-axis#indexed-x-values

File: wpf/Smith-Chart/Series.md [hunk 3]
```diff
@@ -144,3 +144,32 @@ chart.Series.Add(series2);
...
+ 
+The default value is false.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<syncfusion:SfSmithChart x:Name="SmithChart">
+   <syncfusion:LineSeries ArrangeByIndex="False">
```
Comment (vigneshrameshsync) [inline wpf/Smith-Chart/Series.md:159]: By the default value is false, then why should we set it explicitly?

File: wpf/Smith-Chart/Series.md [hunk 4]
```diff
@@ -144,3 +144,32 @@ chart.Series.Add(series2);
...
+
+LineSeries series = new LineSeries();
+series.ArrangeByIndex = false;
+chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Customizing Series Data](Series_images/SmithChart_Series_Image.png)
```
Comment (vigneshrameshsync) [inline wpf/Smith-Chart/Series.md:175]: This is an alternate text for an output image which will be displayed when the image is not get loaded. So, please avoid unnecessary case changes.

Please use the below sample data to capture the output screenshot
https://blazor.syncfusion.com/demos/smith-chart/print-and-export?theme=fluent

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: wpf/Smith-Chart/Series.md [hunk 5]
```diff
@@ -144,3 +144,30 @@ chart.Series.Add(series2);
 {% endtabs %}
 
 ![Series_img3](Series_images/Series_img3.png)
+
+## Customizing data plotting
+
+By default, data points will be sorted and plotted based on the resistance value. By setting the [`ArrangeByIndex`]() property value as true, they can be plotted based on it's index value.
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply wpf/Smith-Chart/Series.md]: I have changed as you mentioned.

File: wpf/Smith-Chart/Series.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply wpf/Smith-Chart/Series.md]: I have corrected.

File: wpf/Smith-Chart/Series.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply wpf/Smith-Chart/Series.md]: I have corrected it.

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

File: wpf/Smith-Chart/Series.md [hunk 6]
```diff
@@ -144,3 +144,29 @@ chart.Series.Add(series2);
 {% endtabs %}
 
 ![Series_img3](Series_images/Series_img3.png)
+
+## Data plotting customization
+
+By default, the data points are plotted by sorting them based on the resistance values and it can be plotted based on their index value as it is by setting the [`ArrangeByIndex`]() property as true.
```
Comment (Chrispineagunja) [inline wpf/Smith-Chart/Series.md:150]: By default, the data points are plotted by sorting them based on the resistance values and they can be plotted based on their index value as it is by setting the [`ArrangeByIndex`]() property as true.

**Comment by TamilarasanGunasekaran** [comment]
rebuild

File: wpf/Smith-Chart/Series.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply wpf/Smith-Chart/Series.md]: I have updated.

**Comment by TamilarasanGunasekaran** [comment]
rebuild

**Pull request merged**